scripts/start

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
+
+set -e
+projectRoot="$(a="/$0"; a=${a%/*}; a=${a:-.}; a=${a#/}/; cd "$a/.." || return; pwd)"
+
+cd "$projectRoot"
+
+docker-compose down --volumes
+docker-compose up --build
+docker-compose down --volumes
```

Comment:
I don't see the value in this script.  If I want to start a service I can just run `docker-compose up --build` and stop it when I want, making a conscious decision to remove the volumes if I want to lose them.

With this approach we destroy all volumes, starts the service then immediately shuts it down destroying the volumes.  If I had data in those volumes I wanted to keep, I'd be frustrated that starting the service wiped all my data.

We also lose the capability to pass additional docker-compose arguments so I don't see why we'd use this and not just Docker Compose.

If a service has a complex start up, then it might warrant a `start` script and teams can make a decision to create their own.  But I'd rather see them optimising their Docker Compose setup so running a service can just use Docker Compose commands.

I don't think we should include this script in a minimal setup repo as the first thing I'd do is delete it.